### DISCUSSION TITLE
阿里网盘报错

### DISCUSSION THREAD

**Discussion by azathoth3** [AlistGo/alist#2302] [Q&A]
添加阿里网盘储存时候显示：
Post "https://auth.aliyundrive.com/v2/account/token": x509: certificate is valid for *.我的域名, 我的域名, not auth.aliyundrive.com

**Comment by azathoth3** [comment]
已经解决了，因为服务器的代理导致证书混乱了

**Reply by Clown12138** [reply]
怎么解决的呢？我刚装好，用了一会儿也出现这个问题了。

**Reply by azathoth3** [reply]
关闭所有代理，重新添加阿里网盘
